arm/Microsoft.Compute/galleries/deploy.bicep

```diff
@@ -68,14 +71,44 @@ resource gallery_lock 'Microsoft.Authorization/locks@2016-09-01' = if (lock != '
 }
 
 module gallery_rbac '.bicep/nested_rbac.bicep' = [for (roleAssignment, index) in roleAssignments: {
-  name: '${deployment().name}-rbac-${index}'
+  name: '${uniqueString(deployment().name, location)}-rbac-${index}'
   params: {
     roleAssignmentObj: roleAssignment
     builtInRoleNames: builtInRoleNames
     resourceName: gallery.name
   }
 }]
 
+// Images
+module galleries_images 'images/deploy.bicep' = [for (image, index) in images: {
+  name: '${uniqueString(deployment().name, location)}-Image-${index}'
+  params: {
+    name: image.name
+    galleryName: gallery.name
+    osType: contains(image, 'osType') ? image.osType : 'Windows'
+    osState: contains(image, 'osState') ? image.osState : 'Generalized'
+    publisher: contains(image, 'publisher') ? image.publisher : 'MicrosoftWindowsServer'
+    offer: contains(image, 'offer') ? image.offer : 'WindowsServer'
+    sku: contains(image, 'sku') ? image.sku : '2019-Datacenter'
+    minRecommendedvCPUs: contains(image, 'minRecommendedvCPUs') ? image.minRecommendedvCPUs : 1
+    maxRecommendedvCPUs: contains(image, 'maxRecommendedvCPUs') ? image.maxRecommendedvCPUs : 4
+    minRecommendedMemory: contains(image, 'minRecommendedMemory') ? image.minRecommendedMemory : 4
+    maxRecommendedMemory: contains(image, 'maxRecommendedMemory') ? image.maxRecommendedMemory : 16
+    hyperVGeneration: contains(image, 'hyperVGeneration') ? image.hyperVGeneration : 'V1'
+    imageDefinitionDescription: contains(image, 'imageDefinitionDescription') ? image.imageDefinitionDescription : ''
+    eula: contains(image, 'eula') ? image.eula : ''
+    privacyStatementUri: contains(image, 'privacyStatementUri') ? image.privacyStatementUri : ''
+    releaseNoteUri: contains(image, 'releaseNoteUri') ? image.releaseNoteUri : ''
+    productName: contains(image, 'productName') ? image.productName : ''
+    planName: contains(image, 'planName') ? image.planName : ''
+    planPublisherName: contains(image, 'planPublisherName') ? image.planPublisherName : ''
+    endOfLife: contains(image, 'endOfLife') ? image.endOfLife : ''
+    excludedDiskTypes: contains(image, 'excludedDiskTypes') ? image.excludedDiskTypes : []
+    roleAssignments: contains(image, 'roleAssignments') ? image.roleAssignments : []
+    tags: contains(image, 'tags') ? image.tags : {}
+  }
+}]
+
 output galleryResourceId string = gallery.id
```

Comment:
Please add `@description('...')` to each. The `Set-ModuleReadMe` function will automatically add them to the readme too